Mathlib/Algebra/Polynomial/Taylor.lean

```diff
@@ -78,6 +80,17 @@ theorem taylor_coeff_zero : (taylor r f).coeff 0 = f.eval r := by
 theorem taylor_coeff_one : (taylor r f).coeff 1 = f.derivative.eval r := by
   rw [taylor_coeff, hasseDeriv_one]
 
+@[simp]
+theorem taylor_coeff_natDegree : (taylor r f).coeff f.natDegree = f.leadingCoeff := by
```

Comment:
Can you first prove `hasseDeriv f.natDegree f = C f.leadingCoeff`? 
Also you should provide a `(taylor r f).leadingCoeff = f.leadingCoeff` version. Thanks!

Reply:
done